test/aleph/http/multipart_test.clj

```diff
@@ -1,20 +1,18 @@
 (ns aleph.http.multipart-test
-  (:use
-   [clojure test])
   (:require
    [aleph.http :as http]
    [aleph.http.multipart :as mp]
    [byte-streams :as bs]
+   [clojure.edn :as edn]
+   [clojure.test :refer [deftest is testing]]
    [manifold.deferred :as d]
-   [manifold.stream :as s]
-   [clojure.string :as str]
-   [clojure.edn :as edn])
+   [manifold.stream :as s])
   (:import
-   [java.io
-    File]))
+   (java.io File)))
 
 (def file-to-send (File. (str (System/getProperty "user.dir") "/test/file.txt")))
 
+#_{:clj-kondo/ignore [:deprecated-var]}
```

Comment:
Almost all tests in this namespace are tagged with this since they all test `mp/encode-body` rather than its apparent successor `mp/encode-request`. Maybe this is an oversight that should be corrected separately?

Reply:
Hmmm. Let's leave this for now.